src/routes/signup.tsx

```diff
@@ -90,16 +102,23 @@ const SignUpForm = () => {
             minLength={minPasswordLength}
             maxLength={maxPasswordLength}
           />
-          <Dropdown<Realm>
-            value={realms.find((r) => r.id === realmId)}
-            emptyLabel="Select a realm"
-            class={dropdownClass}
-            options={realms}
-            required
-            onChange={(v) => setRealmId(v.id)}
-            getKey={(v) => v.id}
-            getText={(v) => v.name}
-          />
+          <div class={realmElementsStyle}>
+            <Dropdown<Realm>
+              value={realms.find((r) => r.id === realmId)}
+              emptyLabel="Select a realm"
+              class={dropdownClass}
+              options={realms}
+              required
+              onChange={(v) => setRealmId(v.id)}
+              getKey={(v) => v.id}
+              getText={(v) => v.name}
+            />
+            <div class={textStyle}>
+              {"Don't see your team? Ask a captain or mentor to "}{' '}
```

Comment:
There is an extra space that gets collapsed here.

```suggestion
              {"Don't see your team? Ask a captain or mentor to "}
```